tests/common/osutil/test_factory.py

```diff
@@ -98,6 +98,13 @@ def test_get_osutil_it_should_return_ubuntu(self):
         self.assertTrue(isinstance(ret, Ubuntu18OSUtil))
         self.assertEqual(ret.get_service_name(), "walinuxagent")
 
+        ret = _get_osutil(distro_name="ubuntu",
+                          distro_code_name="focal",
```

Comment:
Hi, while checking the patches we carry on, I discovered that this should be "noble", not "focal"... I can open an MP/issue for this, but I am making a note here not to forget it.